src/pages/JoinPage.tsx

```diff
@@ -20,6 +21,12 @@ export default function JoinPage() {
 
   const { hasJoinedGame } = useAppSelector(state => state.game);
 
+  // Clear all ccookies if we are joining as a player. This is neccessary
```

Comment:
I think we should also remove all cookies if the game ends. That way the rejoin button won't appear for a game that is already done.

Reply:
Yes this is definitely  something that needs to be implemented. I will add a note to the "End game" issue so that we can handle it once the end game logic is complete.